packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -7071,3 +7071,62 @@ class _StatefulBuilderState extends State<StatefulBuilder> {
   @override
   Widget build(BuildContext context) => widget.builder(context, setState);
 }
+
+/// A widget that paints its area with a specified [Color] and then draws its
+/// child.
+class ColoredBox extends SingleChildRenderObjectWidget {
+  /// Creates a widget that paints its area with the specified [Color].
+  ///
+  /// The `color` parameter must not be null.
```

Comment:
color should be in [] since it exists as a property.

Reply:
Done